builtin/logical/database/path_config_connection.go

```diff
@@ -560,14 +574,49 @@ func (b *databaseBackend) connectionWriteHandler() framework.OperationFunc {
 			oldConn.Close()
 		}
 
+		var performedRotationManagerOpern string
+		if config.ShouldDeregisterRotationJob() {
+			performedRotationManagerOpern = "deregistration"
```

Comment:
I won't suggest we standardize this to the `sdk` because of *waves hands*, but...

Reply:
I genuinely meant we shouldn't do it until a later round, but that's my bad for phrasing.